systemd/systemd.go

```diff
@@ -336,6 +336,8 @@ type Systemd interface {
 	// only necessary to apply manager's configuration like
 	// watchdog.
 	DaemonReexec() error
+	// Enable the given services, optionally with additional arguments
+	Enable(serviceNames []string, additionalArgs []string) error
```

Comment:
Let's see what @pedronis thinks here, but I don't think that the methods of the `Systemd` interface should expose the systemctl parameters like this. I would rather have it take an `EnableOptions` parameter which is a pointer to a struct with the additional information that we need, but without requiring the caller to know how the systemctl options are called. I mean, something like this:

```go
type EnableOptions struct {
    ReloadUnit bool // then we could get rid of the EnableNoReload method, but maybe let's leave this for another PR
    StartNow   bool
}
```

It may be that, since many systemctl invocations accept the same options, we might want to call the struct `{Systemctl,Systemd}Options` and reuse it across different methods. But again, Samuele's call :-)

Reply:
I really like this suggestion, and I considered this shortly except I didn't think of to put 'ReloadUnit' in there. And it felt overkill to have a struct with a single member in.

Reply:
I liked this so much I just went ahead and did this for now.